Sloth.Core/Services/CyberSourceBankReconcileService.cs

```diff
@@ -218,6 +222,26 @@ private async Task ProcessReport(Report report, Integration integration, ILogger
                     tran.Rollback();
                 }
             }
+
+            try
+            {
+                // save copy to blob storage
+                var filename = $"{report.Name}.{report.OrganizationID}.{DateTime.UtcNow:yyyyMMddHHmmssffff}.xml";
+                log.ForContext("container", _options.ReportBlobContainer)
+                    .Information("Uploading {filename} to Blob Storage", filename);
+                await using var memoryStream = new MemoryStream();
+                var writer = new StreamWriter(memoryStream); // lgtm [cs/local-not-disposed]
```

Comment:
Disposing the writer is redundant, because it is just a wrapper for MemoryStream and dispose only flushes and then closes the wrapped stream, both of which are already handled.

Reply:
My vote would be it's worth it because it is disposable and even though it's redundant we shouldn't assume their internal implementation wouldn't change some day.  Not that it will.